frontend/src/components/MapFilters/AttributesFilter/AttributesFilter.tsx

```diff
@@ -0,0 +1,165 @@
+        const opposites = ["abandonedBuildingYes","abandonedBuildingNo","availableYes","availableNo","bicyclesYes","bicyclesNo","","","climbingYes","climbingNo","fieldPuzzleYes","fieldPuzzleNo","","hikeShortYes","hikeShortNo","","nightYes","nightNo","parkingYes","parkingNo","parkngrabYes","parkngrabNo","","","","","teamworkYes","teamworkNo","treeclimbingYes","treeclimbingNo","","","winterYes","winterNo"];
+        const attrArray: Array<AttributeDisplay> = [];
+        for (let i = 0; i < keys.length; i++) {
+            const attr: AttributeDisplay = {
+                name: keys[i],
+                image: images[i],
+                title: titles[i],
+                oppositeName: opposites[i],
+                isDisabled: findStateByName(names[i]),
+                style:  findStyleByName(names[i])
+            };  
+            attrArray.push(attr);
+        }
+        return attrArray;
+    };
+
+    return (
+        <MapFilterItem header="Attribuutit" eventKey={eventKey}>
+            <div className="check-box-filters">
+                {
+                    initAttrArray().map(attr => {
+                        return(
+                            <div className="check-box-filters" key={attr.name}>
+                                <div className={attr.style}>
+                                    <input
+                                        id={attr.name}
+                                        type="checkbox"
+                                        name={attr.name}
+                                        onChange={modifyAttributes}
+                                        data-value={attr.oppositeName}
+                                        disabled= {attr.isDisabled}
+                                        checked={attributes?.[attr.name] || DEFAULT_DISPLAY_VALUE}
+                                    />
+                                    <img src={attr.image} className="icon"/><label htmlFor={attr.name}>{attr.title}</label>
+                                </div>                
+                            </div>
+                        );
+                    })
+                }
+            </div>
+        </MapFilterItem>
+    );
+};
+export default AttributesFilter;
```

Comment:
I checked your translations for the names of the opposite attributes, and I think they are fine.